src/mocks/server.ts

```diff
@@ -0,0 +1,4 @@
+import { setupServer } from 'msw/node';
+import { handlers } from './handlers';
+
+export const server = setupServer(...handlers);
```

Comment:
There is /src/mocks/mswSetup.ts file. Probably this can be included there instead of a new file.